test/semantics/test_errors.sh

```diff
@@ -36,8 +36,12 @@ log=$temp/log
 actual=$temp/actual
 expect=$temp/expect
 diffs=$temp/diffs
+options=$temp/options
 
-cmd="$CMD $src"
+# See if there are additional options
+awk '/^ *!OPTIONS: / {gsub (/^! OPTIONS: /, " " ); print}' $src > $options
```

Comment:
We could be more flexible about allowing spaces and using sed makes this simpler. I suggest:
`sed -n 's/^ *! *OPTIONS: *//p' $src > $options`

Reply:
Thanks!  I'll do it.